edx_sga/static/css/edx_sga.css

```diff
@@ -60,3 +60,69 @@
 .sga-block .error {
     color: red;
 }
+
+.wrapper-comp-settings .list-input.settings-list .field.comp-setting-entry {
```

Comment:
Please prefix all of these with `.modal-type-edx_sga` so we don't style other settings blocks on the same  page